packages/react-i18n/src/i18n.ts

```diff
@@ -316,6 +316,14 @@ export class I18n {
     }).format(date);
   }
 
+  ordinal(amount: number) {
+    const {locale} = this;
+    const group = new Intl.PluralRules(locale, {type: 'ordinal'}).select(
```

Comment:
I'd create the object in the constructor to avoid creating it on every call.

Reply:
Ooh, great point!

Reply:
Done! 🎉